documentation/docs/globals/fetch.mdx

```diff
@@ -41,7 +41,7 @@ Dynamic backends are a compute feature that allow services to define backends fo
 
 When dynamic backends are enabled at the service level, the explicit `backend` option is no longer required for `fetch()` requests, and will instead be automatically created.
 
-In addition, custom backend confiuration options can then also be provided through the [`Backend()`](../fastly:backend/Backend/Backend.mdx) constructor.
```

Comment:
@gaylin is this worth a docs bug bounty? :smile: 
The breadcrumbs start here [fastly documentation](https://www.fastly.com/documentation/guides/compute/javascript/) before going to the SDK Reference

Reply:
Sadly, no. [It's not a typo](https://fastly.atlassian.net/wiki/spaces/DOC/pages/49471718500/Docs+Team+Bug+Bounty+Program) just a mistake. :) But with that said, we officially will make you an honorary tech writer for a day. <3